src/test/java/budgetbuddy/testutil/TypicalPersons.java

```diff
@@ -39,10 +35,10 @@
             .build();
 
     // Manually added - Person's details found in {@code CommandTestUtil}
-    public static final Person AMY = new PersonBuilder().withName(VALID_NAME_AMY)
-            .withTags(VALID_TAG_FRIEND).build();
-    public static final Person BOB = new PersonBuilder().withName(VALID_NAME_BOB)
-            .withTags(VALID_TAG_HUSBAND, VALID_TAG_FRIEND)
+    public static final Person AMY = new PersonBuilder().withName(CommandTestUtil.VALID_NAME_AMY)
+            .withTags(CommandTestUtil.VALID_TAG_FRIEND).build();
+    public static final Person BOB = new PersonBuilder().withName(CommandTestUtil.VALID_NAME_BOB)
+            .withTags(CommandTestUtil.VALID_TAG_HUSBAND, CommandTestUtil.VALID_TAG_FRIEND)
```

Comment:
Might be better to replace these with imports, since that was how it was originally done.
```import static budgetbuddy.logic.commands.CommandTest.Util.VALID_NAME_AMY``` etc.
But it's no big deal.

Reply:
Ahh oops missed those out. I'll change those too